velox/substrait/SubstraitToVeloxPlanValidator.cpp

```diff
@@ -425,6 +481,30 @@ bool SubstraitToVeloxPlanValidator::validate(
       return false;
     }
   }
+
+  if (!validateAggRelFunctionType(sAgg)) {
+    return false;
+  }
+
+  // corner case, groupby and aggregates input is empty
+  if (sAgg.measures_size() == 0) {
+    bool hasExpr = false;
+    for (const auto& grouping : sAgg.groupings()) {
+      for (const auto& groupingExpr : grouping.grouping_expressions()) {
```

Comment:
Maybe `sAgg.groupings().size()` and `grouping.grouping_expressions().size()` can meet the needs here to check whether grouping expression is empty.

Reply:
`sAgg.groupings().size()` may be 1 with empty `grouping_expressions`

Reply:
Can this inner loop `for (const auto& groupingExpr : grouping.grouping_expressions()) {` be removed by using `grouping.grouping_expressions().size()`?